buildspec/release.yaml

```diff
@@ -12,7 +12,7 @@ phases:
       - MINOR_TAG=${PATCH_TAG%.*} # major.minor
       - MAJOR_TAG=${MINOR_TAG%.*} # major
       - docker build -t ${FIPS_REPOSITORY_URI}:${PATCH_TAG} -t ${FIPS_REPOSITORY_URI}:${MINOR_TAG} -t ${FIPS_REPOSITORY_URI}:${MAJOR_TAG} .
-      - docker push --all-tags ${FIPS_REPOSITORY_URI}:${PATCH_TAG}
+      - docker push --all-tags ${FIPS_REPOSITORY_URI}
```

Comment:
I wouldn't have thought that we needed the patch tag specification with the `--all-tags` arg 🤔

Reply:
oh, I see your message about codebuild now, nvm